src/replication.c

```diff
@@ -190,6 +198,76 @@ void rebaseReplicationBuffer(long long base_repl_offset) {
     }
 }
 
+/* Replication: Primary side - connections association.
+ * On rdb-connection sync, connection association is used to keep replication data in 
+ * the backlog until the replica requests PSYNC. Association happens in two forms, 
+ * if there's an existing buffer block at the fork time, the replica is attached
+ * to the tail, if there is no tail, the replica will be attached when a new
+ * buffer block is created (see the Retrospect function below).
+ * Replica rdb client id is used as a unique key for the association. 
+ * On COB overrun, association is deleted and the RDB connection 
+ * is dropped.
+ */
+void addReplicaToPsyncWaitingRax(client* replica) {
```

Comment:
Rax is an implementation detail and could change in the future.
```suggestion
void addReplicaToPsyncWait(client* replica) {
```